erpnext/setup/demo.py

```
@@ -16,21 +16,39 @@
 
 	capture("demo_data_creation_started", "erpnext")
 	try:
-		company = create_demo_company()
+		company = create_demo_company(company_name)
 		process_masters()
 		make_transactions(company)
-		frappe.cache.delete_keys("bootinfo")
-		frappe.publish_realtime("demo_data_complete")
+		capture("demo_data_creation_completed", "erpnext")
+		frappe.clear_messages()
 	except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)